opentelemetry-aws/Cargo.toml

```diff
@@ -31,5 +31,5 @@ opentelemetry_sdk = { workspace = true, optional = true }
 opentelemetry_sdk = { workspace = true, features = ["testing"] }
```

Comment:
given the update to these crates, we need a changelog entry. or force them to use 0.23 itself

Reply:
Oh yes, there were few more which actually don't need any change. Updated their changelog.